docs/docs/about.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: About
+permalink: /about
+nav_order: 2
+---
+
+## About OCI
+
+The Open Container Initiative was established in June 2015 by 
```

Comment:
some sections like this could lean on wording from https://www.opencontainers.org/

Reply:
I didn't want to copy verbatim, so I put in my own words. Perhaps someone that has better knowledge of the actual history / about could work on this after?

Reply:
This is definitely resolved, I added "and other leaders in the container industry"